components/FeatureHighlight/tests/unit/FeatureHighlightTitleBodyAccessibilityMutatorTests.m

```diff
@@ -0,0 +1,141 @@
+      (MDCFeatureHighlightView *)featureHighlightViewController.view;
+
+  // Making the background color the same as the title/body color.
+  featureHighlightView.outerHighlightColor = [UIColor whiteColor];
+  featureHighlightView.titleColor = [UIColor whiteColor];
+  featureHighlightView.bodyColor = [UIColor whiteColor];
+  [MDCFeatureHighlightAccessibilityMutator mutate:featureHighlightViewController];
+
+  XCTAssertNotNil(featureHighlightView.titleColor);
+  XCTAssertNotNil(featureHighlightView.bodyColor);
+
+  // Hard coded values, if this test fails it only means that we could mean that we changed the
+  // accessible color we are returning.
+  XCTAssertEqualObjects(featureHighlightView.titleColor,
+                        [UIColor colorWithWhite:0 alpha:0.87000000476837158]);
+  XCTAssertEqualObjects(featureHighlightView.bodyColor,
+                        [UIColor colorWithWhite:0 alpha:0.54000002145767212]);
+}
+
+- (void)testMutatorKeepsAccessibleTextColor {
+  NSDictionary* colors = @{ [UIColor redColor]: [UIColor blackColor]};
+  for (UIColor *color in colors) {
+    MDCFeatureHighlightViewController *featureHighlightViewController =
+        [[MDCFeatureHighlightViewController alloc] initWithHighlightedView:self.highlightedView
+                                                               andShowView:self.showView
+                                                                completion:nil];
+    MDCFeatureHighlightView *featureHighlightView =
+        (MDCFeatureHighlightView *)featureHighlightViewController.view;
+
+    // Making the background color accessible with title/body color.
+    featureHighlightView.outerHighlightColor = colors[color];
+    featureHighlightView.titleColor = color;
+    featureHighlightView.bodyColor = color;
+
+    [MDCFeatureHighlightAccessibilityMutator mutate:featureHighlightViewController];
+
+    XCTAssertEqualObjects(featureHighlightView.titleColor, color);
+    XCTAssertEqualObjects(featureHighlightView.bodyColor, color);
+  }
+}
+
+- (void)testMutatorSelectsTheRightColorWhenThereIsNoColorSet {
+  MDCFeatureHighlightViewController *featureHighlightViewController =
+      [[MDCFeatureHighlightViewController alloc] initWithHighlightedView:self.highlightedView
+                                                             andShowView:self.showView
+                                                              completion:nil];
+  MDCFeatureHighlightView *featureHighlightView =
+      (MDCFeatureHighlightView *)featureHighlightViewController.view;
+
+  // Making the background color accessible with title/body color.
+  featureHighlightView.outerHighlightColor = [UIColor blackColor];
+
+  [MDCFeatureHighlightAccessibilityMutator mutate:featureHighlightViewController];
+
+  XCTAssertNotNil(featureHighlightView.titleColor);
+  XCTAssertNotNil(featureHighlightView.bodyColor);
+  XCTAssertNotEqualObjects(featureHighlightView.titleColor, [UIColor blackColor]);
```

Comment:
I'm concerned that we're testing that it isn't `blackColor`.  Colors like `clearColor` should not be considered "passing".